test/e2e/app-dir/ppr-unstable-cache/ppr-unstable-cache.test.ts

```diff
@@ -53,13 +53,13 @@ describe('ppr-unstable-cache', () => {
       env: { TEST_DATA_SERVER: `http://localhost:${port}/` },
     })
 
-    expect(generations).toHaveLength(3)
+    expect(generations).toHaveLength(2)
```

Comment:
This test is expected to change because it was previously including the extra render in the assertions.